util/cosmos/iscn/sign.js

```diff
@@ -9,11 +9,10 @@ import {
 } from '@cosmjs/stargate';
 import jsonStringify from 'fast-json-stable-stringify';
 
-import { DEFAULT_GAS_PRICE_NUMBER } from '../../CosmosHelper';
+import { getGasLimitParams } from '@/util/cosmos/iscn/gasEstimator';
```

Comment:
Should just return the result of the linear regression as hardcode constant in this repo to save computation power of client